core/templates/dev/head/player/conversation_skin_directive.html

```diff
@@ -37,10 +37,150 @@
         cards, (b) the screen is wide, in which case both cards appear
         side-by-side.
       -->
+      <div class="conversation-skin-cards-container-inside">
+      
+        <div class="conversation-skin-left-card-container">
+                  <div class="conversation-skin-main-tutor-card conversation-skin-tutor-card-active">
```

Comment:
Something seems to have gone a bit wrong in the indentation here (which may be what's making it hard to see the diff). It's jumped by 10 spaces, whereas it should differ from the previous line by only 2. Please could you fix this?